pkg/shell/nav.jsx

```diff
@@ -123,7 +123,7 @@ export class CockpitNav extends React.Component {
 
         return (
             <>
-                <SearchInput placeholder={_("Search")} value={this.state.search} onChange={search => this.setState({ search })} onClear={() => this.setState({ search: "" })} className="search" />
+                <SearchInput placeholder={_("Search")} value={this.state.search} onChange={(_, search) => this.setState({ search })} onClear={() => this.setState({ search: "" })} className="search" />
                 <Nav onSelect={this.onSelect} theme="dark">
                     { groups.map(g =>
                         <section className="pf-c-nav__section" aria-labelledby={"section-title-" + g.name} key={g.name}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18222-20230124-102908-ad85759e-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)